runtimes/pythonrt/local_runner.go

```diff
@@ -145,6 +144,32 @@ func (r *LocalPython) Start(pyExe string, tarData []byte, env map[string]string,
 }
 
 func (r *LocalPython) Health() error {
+	if _, err := os.FindProcess(r.proc.Pid); err != nil {
+		return fmt.Errorf("runner heath: no runner proc found. %w ", err)
+	}
+
+	var status syscall.WaitStatus
+	if pid, err := syscall.Wait4(r.proc.Pid, &status, syscall.WNOHANG, nil); err == nil && pid == r.proc.Pid {
+		if status.Signaled() {
+			sig := status.Signal()
+			switch sig {
+			case syscall.SIGKILL, syscall.SIGTERM, syscall.SIGABRT, syscall.SIGSEGV, syscall.SIGBUS:
+				err = fmt.Errorf("runner health: proc terminated by signal %v", sig)
```

Comment:
Add pid?

Reply:
meanwhile pid is not relevent info. We are not trackign it neither on start, not on stop.
we could tho, but adding it only here is feels redudndant for me meanwhile